bot.py

```diff
@@ -43,6 +43,12 @@ async def on_ready():
 
 @bot.check
 async def block_disabled_commands(ctx):
+    result = (await selector("SELECT config FROM settings WHERE GUILD = %s", [ctx.guild.id]))
+    if result == ():
+        configuration = {'currency': True, 'socials': True}
+        configuration = json.dumps(configuration)
```

Comment:
Why not json.dumps that dictionary right away?

Reply:
Because I don't want to do that.

Reply:
I agree with not dumping it right away, as seeing function calls in a part where inputs should be makes it longer (which is bad) and more ugly